modules/standard/Math.chpl

```diff
@@ -1056,6 +1056,19 @@ module Math {
     extern proc truncf(x: real(32)): real(32);
     return truncf(x);
   }
+  
+  /* Returns the greatest common divisor of the integer argument `arg1` and
+     `arg2`. */
+  proc gcd(in a: int,in b: int): int {
+     var r: int;
+     while(b != 0) {
+       r = a % b;
```

Comment:
I have some discomfort with `%` applied to negative numbers. Would you mind taking abs of a and b before the loop, instead of taking the abs when returning?

Reply:
Okay sir !! I will update this though it do not have much affect . But I think your suggestion is good . I will follow it. Thanks !! 😊